pype/plugins/global/publish/integrate_new.py

```diff
@@ -520,23 +524,6 @@ def integrate(self, instance):
             self.log.debug("Hardlinking file .. {} -> {}".format(src, dest))
             self.hardlink_file(src, dest)
 
-    def unc_convert(self, path):
```

Comment:
Has anybody idea, Why the hell this was here?

Reply:
only for converting environment that we were sending originally. it's is not needed anymore thanks to pype app launcher on the farm

Reply:
actually also very old legacy, where we were still using hardcoded path in represenation